Transistor_FET.dcm

```diff
@@ -2377,146 +2377,152 @@ F https://www.infineon.com/dgdl/irf9383mpbf.pdf?fileId=5546d462533600a4015356116
-K Dual N-Channel MOSFET
+D 15A Id, 20V Vds, 4.9mOhm Rds, Dual N-Channel HEXFET Power MOSFET, DirectFET SA
+K Dual N-Channel HEXFET MOSFET Logic-Level
 F https://www.infineon.com/dgdl/irl6297sdpbf.pdf?fileId=5546d462533600a40153565ff22f2575
 $ENDCMP
 #
 $CMP IRL7472L1
-D 375A Id, 40V Vds, 0.45mOhm Rds, N-Channel MOSFET, DirectFET L8
-K N-Channel MOSFET
+D 375A Id, 40V Vds, 0.45mOhm Rds, N-Channel HEXFET Power MOSFET, DirectFET L8
+K N-Channel HEXFET MOSFET Logic-Level IRFET
 F https://www.infineon.com/dgdl/Infineon-IRL7472L1-DS-v02_00-EN.pdf?fileId=5546d46254e133b401555d17178250d8
 $ENDCMP
 #
 $CMP IRLB8721PBF
-D 62A Id, 30V Vds, N-Channel MOSFET, TO-220
-K N-Channel HEXFET Power MOSFET
+D 62A Id, 30V Vds, 8.7 mOhm Rds, N-Channel HEXFET Power MOSFET, TO-220
+K N-Channel HEXFET MOSFET Logic-Level
 F http://www.infineon.com/dgdl/irlb8721pbf.pdf?fileId=5546d462533600a40153566056732591
 $ENDCMP
 #
 $CMP IRLIZ44N
-D 30A Id, 55V Vds, Single N-Channel HEXFET Power MOSFET, 22mOhm Ron, TO-220AB
-K Single N-Channel HEXFET Power MOSFET
+D 30A Id, 55V Vds, 22mOhm Rds, N-Channel HEXFET Power MOSFET, TO-220AB
+K N-Channel HEXFET MOSFET Logic-Level
 F http://www.irf.com/product-info/datasheets/data/irliz44n.pdf
 $ENDCMP
 #
 $CMP IRLML0030
-D 5.3A Id, 30V Vds, N-Channel HEXFET Power MOSFET, 27mOhm Rds, SOT-23
-K N-Channel HEXFET Power MOSFET
+D 5.3A Id, 30V Vds, 27mOhm Rds, N-Channel HEXFET Power MOSFET, SOT-23
+K N-Channel HEXFET MOSFET Logic-Level
 F https://www.infineon.com/dgdl/irlml0030pbf.pdf?fileId=5546d462533600a401535664773825df
 $ENDCMP
 #
 $CMP IRLML2060
-D 1.2A Id, 60V Vds, N-Channel MOSFET, 480mOhm Rds, SOT-23
-K N-Channel MOSFET
+D 1.2A Id, 60V Vds, 480mOhm Rds, N-Channel HEXFET Power MOSFET, SOT-23
+K N-Channel HEXFET MOSFET Logic-Level
 F https://www.infineon.com/dgdl/irlml2060pbf.pdf?fileId=5546d462533600a401535664b7fb25ee
 $ENDCMP
 #
 $CMP IRLML5203
-D -3.0A Id, -30V Vds, P-Channel HEXFET Power MOSFET, 98mOhm Rds, SOT-23
-K P-Channel MOSFET
+D -3.0A Id, -30V Vds, 98mOhm Rds, P-Channel HEXFET Power MOSFET, SOT-23
+K P-Channel HEXFET MOSFET Logic-Level
 F https://www.infineon.com/dgdl/irlml5203pbf.pdf?fileId=5546d462533600a40153566868da261d
 $ENDCMP
 #
+$CMP IRLML6401
+D -4.3A Id, -12V Vds, 50mOhm Rds, P-Channel HEXFET Power MOSFET, SOT-23
+K P-Channel HEXFET MOSFET Logic-Level
+F https://www.infineon.com/dgdl/irlml6402pbf.pdf?fileId=5546d462533600a401535668d5c2263c
+$ENDCMP
+#
 $CMP IRLML6402
-D -3.7A Id, -20V Vds, P-Channel MOSFET, 65mOhm Rds, SOT-23
-K P-Channel MOSFET
+D -3.7A Id, -20V Vds, 65mOhm Rds, P-Channel HEXFET Power MOSFET, SOT-23
+K P-Channel HEXFET MOSFET Logic-Level
 F https://www.infineon.com/dgdl/irlml6402pbf.pdf?fileId=5546d462533600a401535668d5c2263c
 $ENDCMP
 #
 $CMP IRLML9301
-D -3.6A Id, -30V Vds, P-Channel HEXFET Power MOSFET, 64mOhm Rds, SOT-23
-K P-Channel MOSFET
+D -3.6A Id, -30V Vds, 64mOhm Rds, P-Channel HEXFET Power MOSFET, SOT-23
+K P-Channel HEXFET MOSFET Logic-Level
```

Comment:
Same as above, I could not find the word `logic in the datasheet.

Reply:
Might be true. Product webpage claims it's "logic level optimized" for 4.5 and 2.5V
https://www.infineon.com/cms/en/product/power/mosfet/12v-300v-n-channel-power-mosfet/irlml9301/